docs/ml/toolkit/index.md

```diff
@@ -1,74 +1,73 @@
 ---
```

Comment:
This file seems like it was overhauled from `docs/ml/index.md`? Are these changes all as expected?